lib/QubitFlatfileImport.class.php

```diff
@@ -37,6 +37,7 @@ class QubitFlatfileImport
   public $deleteAndReplace       = false; // Delete matching records & replace them
   public $skipMatched            = false; // Skip creating new record if matching one is found
   public $skipUnmatched          = false; // Skip creating new record if matching one is not found
+  public $limitToId              = 0;     // Id of repository or TLD to limit our update matching under
```

Comment:
Maybe filterByRepositoryId would be clearer. Limit makes me think of skip/limit in queries and I'd assume that "Id" is referencing the ID of a matched object rather than its repository.